cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -126,6 +126,10 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 			parent.Close()
 			return nil, err
 		}
+		if err := oci.HandleCPUGroupSetup(ctx, parent, s.Annotations); err != nil {
```

Comment:
I don't like passing in the UVM into the oci package and having it do work against it. Can we just cache the `processorTopology` information we get when we call `HostProcessorInfo` during UVM setup and just pass this into an exported `AnnotationsToCpuGroupOptions` and have this return the info we need?